sdk/core/core-lro/test/engine.spec.ts

```diff
@@ -543,7 +533,6 @@ describe("Lro Engine", function () {
             routes: [
               {
                 method: "GET",
-                path: `/LROPostDoubleHeadersFinalLocationGet`,
```

Comment:
The main reason I made this change is because when I created this unit test recently, I copied it from another and the value for the path property suggests that the initial request method is a POST but it is in fact a GET. Either way, the path value doesn't add any value to the test anymore and worse it could be misleading as it is the case for this test.